index.html

```diff
@@ -177,6 +177,54 @@ <h3 class="center">Our work reflected in numbers</h3>
   </div>
 </div>
 
+<div class="section teal">
+  <div class="container row">
+    <h3 class="center">Partners and Friends of FCCVienna</h3>
+    <p class="center">If you enjoyed taking part in one of our meetings, and you feel like connecting to other people, these partners are also organizing amazing groups.</p>
+  </div>
+  <div class="container row">
+    <div class="col s12 m3 offset-m3">
+      <a class="partner-logo" href="http://www.webclerks.at/">
+        <img class="center-block" src="./images/webclerks-logo.jpeg" alt="webclerks">
+      
+      <p class="center">Webclerks</p>
+      </a>
+    </div>
+    <div class="col s12 m3">
+      <a class="partner-logo" href="https://reactiveconf.com/">
+        <img class="center-block" src="./images/reactive-conf-logo.jpg" alt="reactive conf">
+      <p class="center">ReactiveConf</p>
+      </a>
+    </div>
+  </div>
+</div>
+
+<div class="section teal darken-1">
+  <div class="container row">
+    <h3 class="center">Contributors</h3>
+    <p class="center">These are the people who have contributed actively more than once with content for our meetings. Without these creators, there would be no FCCVienna.</p>
+    <div class="col s12 m4 offset-m2">
+      <ul class="contributors center">
+        <li><a href="https://github.com/MatthiasLiszt">Mathias Liszt</a></li>
```

Comment:
The default font-size for these are 1.3em, while the font-size for <p> elements like name of partners are 1.2em. How about we normalize this by adding a class to li elements which sets them to 1.2em?
.li-normalize {
  font-size: 1.2em;
}

Reply:
What about adding a a class to the ul, and then styling its li elements:
.ul-normalize {
font-size: 1.2em;
}
 I think it would make it better for iteration later on, than manually adding the class to every entry on the list.

Reply:
sounds perfect to me! :+1: